x-pack/plugins/observability/server/saved_objects/slo.ts

```diff
@@ -27,35 +27,35 @@ export const slo: SavedObjectsType = {
           params: { type: 'flattened' },
         },
       },
-      time_window: {
+      timeWindow: {
```

Comment:
Out of curiosity, aren't these filed coming from Elasticsearch?

Reply:
It's the mapping of saved object for the "SLO" type. Behind the scene, the data is stored in Elasticsearch